app/views/registration/_form.erb

```diff
@@ -0,0 +1,71 @@
+<% title _("Register Host") %>
+<%= form_tag({controller: 'registration', action: 'create' }, method: :post, class: 'form-horizontal well') do %>
+  <div class="form-group ">
+    <label class="col-md-2 control-label"><%= _('Organization') %></label>
+    <div class="col-md-4">
+      <%= select_tag 'organization', options_from_collection_for_select(@organizations, :id, :name, params[:organization]), class: 'form-control' %>
+    </div>
+  </div>
+  <div class="form-group ">
+    <label class="col-md-2 control-label"><%= _('Location') %></label>
+    <div class="col-md-4">
+      <%= select_tag 'location', options_from_collection_for_select(@locations, :id, :name, params[:location]), class: 'form-control' %>
+    </div>
+  </div>
+  <div class="form-group ">
+    <label class="col-md-2 control-label"><%= _('Host Group') %></label>
+    <div class="col-md-4">
+      <%= select_tag 'host_group', options_from_collection_for_select(@host_groups, :id, :name, params[:host_group]), class: 'form-control', include_blank: '' %>
+    </div>
+  </div>
+  <div class="form-group ">
+    <label class="col-md-2 control-label"><%= _('Operating System') %></label>
+    <div class="col-md-4">
+      <%= select_tag 'operating_system', options_from_collection_for_select(@operating_systems, :id, :title, params[:operating_system]), class: 'form-control', include_blank: '' %>
+    </div>
+  </div>
+  <div class="form-group ">
+    <label class="col-md-2 control-label"><%= _('Proxy') %></label>
+    <div class="col-md-4">
+      <%= select_tag 'smart_proxy', options_from_collection_for_select(@smart_proxies, :id, :name, params[:smart_proxy]), class: 'form-control', include_blank: '' %>
+    </div>
+  </div>
+  <div class="form-group ">
+    <label class="col-md-2 control-label"><%= _('JWT Expiration') %>&nbsp;<%= popover('', _('Expiration of JWT authorization token in hours'), options = {}) %></label>
+    <div class="col-md-4">
+      <%= number_field_tag 'jwt_expiration', 4, class: 'form-control', min: 1 %>
+    </div>
```

Comment:
I do believe bootstrap can have a suffix field where you can place the `hours` bit, but it's been a while since I did any UI work.

Reply:
I didn't find anything, can you give me a link?